test/wasm/library_config/README.md

```diff
@@ -0,0 +1,11 @@
+# Libconfig example
+
+## How to run
```

Comment:
Could this be done through a `package.json` script?

Reply:
Also, any way to always build WASM addons at the same time as native ones? This would remove the need to think about specific crates when building the whole library.

Reply:
Yea definitely, I put it into the top level package.json in [0ee74a1](https://github.com/DataDog/libdatadog-nodejs/pull/44/commits/0ee74a1ae810716f4f546446adc1a1086394b76b). 

> Also, any way to always build WASM addons at the same time as native ones?

It's done when using yarn; but for the CI I'd rather not, `prebuildify` doesn't make sense for such a simpl cross-arch compilation task.